drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -675,6 +675,32 @@ static int stm32_clock_control_init(const struct device *dev)
 	/* Update CMSIS variable */
 	SystemCoreClock = CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC;
 
+#if defined(CONFIG_CLOCK_STM32_PLL2_ENABLE)
+	/* Configure PLL2 */
+	/* Select clock source */
+	/* Init pre divider DIVM */
+	LL_RCC_PLL2_SetM(CONFIG_CLOCK_STM32_PLL2_M_DIVISOR);
+
+	/* Config PLL */
+	/* VCO sel, VCO range */
+	LL_RCC_PLL2_SetVCOInputRange(LL_RCC_PLLINPUTRANGE_1_2);
+	LL_RCC_PLL2_SetVCOOutputRange(LL_RCC_PLLVCORANGE_WIDE);
+
+	/* FRACN disable DIVP,DIVQ,DIVR enable*/
+	LL_RCC_PLL2FRACN_Disable();
+	LL_RCC_PLL2P_Enable();
+	LL_RCC_PLL2Q_Enable();
+	LL_RCC_PLL2R_Enable();
```

Comment:
It should be possible to enable channels individually, if I'm not wrong SDMMC only needs R if using PLL2. I have pushed support PLL3 in https://github.com/zephyrproject-rtos/zephyr/pull/30906, it may be useful

Reply:
So possibly all PLLs should be configurable the same way with Kconfig. Then it makes sense to start adding clock source selectors for all drivers? So far I have not found it in any...

Reply:
> So possibly all PLLs should be configurable the same way with Kconfig. Then it makes sense to start adding clock source selectors for all drivers? So far I have not found it in any...

This is indeed something that will be requested, but it doesn't make much sense to add it right now using Kconfig. Let's live without it few months more and get this using dts.